prod.env

```diff
@@ -1,9 +1,5 @@
-# Domain
-DOMAIN=localhost
-
-# Environment: local, staging, production
-ENVIRONMENT=local
-PROJECT_NAME="TAD"
+# Environment: local, production, demo
+ENVIRONMENT=production
```

Comment:
if it is for demo purposes and we don't have an acceptance yet, wouldn't staging be better then production?

Reply:
i think staging should run as production. Often you want staging to run similarly to production. so i do not think it makes sense to make a differentiation in the settings for the application. We could even consider removing the ENVIRONMENT totally because it is only used to check if the parameters are allowed in some cases